src/main/java/com/spotinst/sdkjava/model/bl/ocean/ecs/OceanEcsCluster.java

```diff
@@ -0,0 +1,252 @@
+package com.spotinst.sdkjava.model.bl.ocean.ecs;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.Date;
+import java.util.Set;
+import java.util.HashSet;
+
+/**
+ * Created by Bansi Chapla on 20/07/2021.
+ */
+
+public class OceanEcsCluster {
+
+    @JsonIgnore
+    private Set<String>                            isSet;
+    private String                                 id;
```

Comment:
Is this field needed.

Reply:
Yes, Id is needed. we are setting this id with Id we get in cluster creation and then using same id for update, delete operation.

Reply:
Yes, To get cluster id this field is required.